src/rshim_fuse.c

```diff
@@ -893,6 +907,10 @@ static int rshim_fuse_misc_write(struct cuse_dev *cdev, int fflags,
     bd->has_cons_work = 1;
     rshim_work_signal(bd);
     pthread_mutex_unlock(&bd->mutex);
+  } else if (!strcmp(key, "OPN_STR")) {
+    if (sscanf(p, "%16s", opn) != 1)
```

Comment:
don't we need a check here for RSHIM_BLUEFIELD_2 cuz this isn't supported in BF1?